### PR TITLE
Doc update for SPN control at subscription and RG scope

### PR DESCRIPTION
**Pull request by v-rahkuma**
Updated doc for Azure_Subscription_AuthZ_Dont_Grant_SPNs_Privileged_Roles, Azure_Subscription_AuthZ_Dont_Grant_SPNs_Privileged_Roles_RG

### CODE REVIEW

File: Control coverage/Feature/SubscriptionCore.md [hunk 1]
```diff
@@ -22,7 +22,8 @@
 - [Azure_Subscription_AuthZ_Limit_Admin_Owner_Count](#Azure_Subscription_AuthZ_Limit_Admin_Owner_Count)
 - [Azure_Subscription_SI_Dont_Use_B2C_Tenant](#azure_subscription_si_dont_use_b2c_tenant)
 - [Azure_Subscription_AuthZ_Dont_Use_SPNs_With_Password](#Azure_Subscription_AuthZ_Dont_Use_SPNs_With_Password)
-
+- [Azure_Subscription_AuthZ_Dont_Grant_SPNs_Privileged_Roles](#Azure_Subscription_AuthZ_Dont_Grant_SPNs_Privileged_Roles)
+- [Azure_Subscription_AuthZ_Dont_Grant_SPNs_Privileged_Roles_RG](#Azure_Subscription_AuthZ_Dont_Grant_SPNs_Privileged_Roles_RG)
```
Comment (asimks) [inline Control coverage/Feature/SubscriptionCore.md:26]: Not coming up as link. Please check

File: Control coverage/Feature/SubscriptionCore.md [hunk 2]
```diff
@@ -1116,4 +1117,88 @@ The purpose of the security control is to prevent the creation of Service Princi
...
+>
+
+### Recommendation 
+
+- If these SPNs need access to your subscription, make sure you add them at the specific permission scope and role required for your scenario. For example, sometimes 'Contributor' access at 'Resource Group' scope might be sufficient. In other scenarios you may need 'Reader' access at 'Subscription' scope. Exact permission will vary based on your use case.
+
+- **PowerShell** 
+
+	 ```powershell	 
+        # Below commands will be useful to Read and Remove Password Credentials from Service Principals
```
Comment (asimks) [inline Control coverage/Feature/SubscriptionCore.md:1146]: Description seems to be incorrect

File: Control coverage/Feature/SubscriptionCore.md [hunk 3]
```diff
@@ -1116,4 +1117,88 @@ The purpose of the security control is to prevent the creation of Service Princi
...
```
Comment (asimks) [inline Control coverage/Feature/SubscriptionCore.md:1188]: Description seems to be incorrect

File: Control coverage/Feature/SubscriptionCore.md (hunk 2 above)
Comment (v-rahkuma) [reply Control coverage/Feature/SubscriptionCore.md]: Done, updated the description to # Below commands will be useful to remove Service Principals

File: Control coverage/Feature/SubscriptionCore.md (hunk 1 above)
Comment (v-rahkuma) [reply Control coverage/Feature/SubscriptionCore.md:26]: As discussed it is coming as link, marking the comment as closed

File: Control coverage/Feature/SubscriptionCore.md (hunk 3 above)
Comment (v-rahkuma) [reply Control coverage/Feature/SubscriptionCore.md]: Done, updated the description to # Below commands will be useful to remove Service Principals

File: Control coverage/Feature/SubscriptionCore.md [hunk 4]
```diff
@@ -1116,4 +1117,88 @@ The purpose of the security control is to prevent the creation of Service Princi
...
+>
+
+### Recommendation 
+
+- If these SPNs need access to your subscription, make sure you add them at the specific permission scope and role required for your scenario. For example, sometimes 'Contributor' access at 'Resource Group' scope might be sufficient. In other scenarios you may need 'Reader' access at 'Subscription' scope. Exact permission will vary based on your use case.
+
+- **PowerShell** 
+
+	 ```powershell	 
+        # Below commands will be useful to remove Service Principals
```
Comment (asimks) [inline Control coverage/Feature/SubscriptionCore.md:1146]: It is still not correct Rahul.
It should say - Below commands will be useful to remove privileged role assignments for SPNs

File: Control coverage/Feature/SubscriptionCore.md [hunk 5]
```diff
@@ -1116,4 +1117,88 @@ The purpose of the security control is to prevent the creation of Service Princi
...
```
Comment (asimks) [inline Control coverage/Feature/SubscriptionCore.md:1188]: Same as above.

File: Control coverage/Feature/SubscriptionCore.md (hunk 4 above)
Comment (v-rahkuma) [reply Control coverage/Feature/SubscriptionCore.md]: Updated the des as suggested, Below commands will be useful to remove privileged role assignments for SPNs

File: Control coverage/Feature/SubscriptionCore.md (hunk 5 above)
Comment (v-rahkuma) [reply Control coverage/Feature/SubscriptionCore.md]: Updated the des as suggested, Below commands will be useful to remove privileged role assignments for SPNs

**Review by asimks — APPROVED** [review]

**Review by Aboli-msft — APPROVED** [review]

**Pull request merged**